docs/SQLModelingStandard.md

```diff
@@ -76,7 +76,7 @@ infrastructure will be re-designed to follow this guideline.*
   be named like in `OriginCountryID` and `DestinationCountryID`.*
 - Both **primary key and foreign key** fields should end with **`ID`**
   in upper case.  
-- **Date** fields should not just be named “Date”. A
+- **Date** fields should not be named “Date”. A
```

Comment:
this use of `just` and the one above are not condescending.  their synonyms here are "only" and the support the logical construct.  I don't agree with stripping them here.